### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by amirtha-coder**
**Work Done**
- [ ] HTML semantic elements added
- [ ] Navigation bar added
- [ ] About me, projects and other projects created 
- [ ] Footer added
- [ ] Images and links added

**TODO**
 - [ ]  Refactor CSS to make it more polished and reduce the likelihood of bugs.
 - [ ] add animations to CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+
+## Deployed URL 
+
+Click the link [here](https://amirtha-coder.github.io/amirtha-portfolio/) to access the webpage.
+
+## Previous projects
+
+- #### Project 1: semantic HTML and adding CSS to refactor the webpage for a marketing agency. Click the link [here](https://winner-am1.github.io/semantic_html_refactor/) to access the webpage.
+- ...*PENDING*...
+-  ...*PENDING*...
```
Comment (LeeFarnell) [inline README.md:52]: As mentioned in the feedback, please ensure you add screenshots to your ReadMe.

File: assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+float: left;
+
+}
+
+.contact-image {
+height:50px;
+width: 50px;
+padding: 15px;
+margin: 10px;
+
```
Comment (LeeFarnell) [inline assets/css/styles.css]: I think these could be bigger, it would make them clearer to the user.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+          </div>
+    
+      </section>
+    </main>
+    <footer class="footer">
+        <h3 class="hr-lines">&#127881; End &#127881; </h3>
+        <p> Thank you for visiting my portfolio </p>
+    </footer>
+  </body>
+</html>
```
Comment (LeeFarnell) [inline index.html]: This is all good. Really good use of semantic elements. well done.

File: assets/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+  margin-bottom: 10vh;
+}
+
+.project-items {
+  display: flex;
+  flex-direction: row;
+  border-left: 5px solid black;
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+  height: 30rem;
+  width: 40vw;
```
Comment (chelseanicholls) [inline assets/css/styles.css:137]: A fixed width on the projects causes them to look very narrow on a mobile viewport. Add the width to media queries to fix this.

**Pull request merged**